### PR TITLE
Imports benbjohnson/testing and use it in web package

### PR DESCRIPTION
**Pull request by jlevesy**
This PR refers to #3242 

In this PR I:

- [x] Imported benbjohnson/testing helpers in the `util/testutil` pacakage
- [x] Used those helpers in the `web` package testsuite

This is a quick PR just to make sure I'm in the right direction. Feel free to comment. 

ping @Gouthamve

### CODE REVIEW

File: web/web_test.go [hunk 1]
```diff
@@ -99,55 +99,37 @@ func TestReadyAndHealthy(t *testing.T) {
...
 	resp, err = http.Get("http://localhost:9090/version")
-	if err != nil {
-		t.Fatalf("Unexpected HTTP error %s", err)
-	}
-	if resp.StatusCode != http.StatusServiceUnavailable {
-		t.Fatalf("Path /version with server unready test, Expected status 503 got: %s", resp.Status)
-	}
+
+	testutil.Ok(t, err)
+	testutil.Equals(t, resp.StatusCode, http.StatusServiceUnavailable)
```
Comment (Gouthamve) [inline web/web_test.go]: I think Equals is `(t, expected, actual)`

**Comment by alcortesm** [comment]
Travis complains because the new file util/testutil/testing.go has no license text at the beginning.

The project `benbjohnson/testing` is MIT, so I think just adding the text in https://github.com/benbjohnson/testing/blob/master/LICENSE as a comment at the beginning will be OK.

File: web/web_test.go (hunk 1 above)
Comment (jlevesy) [reply web/web_test.go]: D'uh, went too fast, sorry about that.

**Comment by fabxc** [comment]
Thanks 👍  :)

**Pull request merged**